src/api/system_api.js

```diff
@@ -459,6 +459,13 @@ module.exports = {
             auth: {
                 system: 'admin'
             }
+        },
+
+        get_system_store: {
+            method: 'GET',
+            auth: {
```

Comment:
Better to return the `RPC_BUFFERS` under a reply object, like in [block_store_api](https://github.com/alphaprinz/noobaa-core/blob/766302a61a550aef4b63b4a02da18ed10d3ec711/src/api/block_store_api.js#L191-L201) for example.


```suggestion
            reply: {
                type: 'object',
                properties: {
                    // [RPC_BUFFERS].data
                },
            },
            auth: {
```